Velaptor/Observables/OpenGLInitReactor.cs

```diff
@@ -0,0 +1,39 @@
+﻿// <copyright file="OpenGLInitReactor.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace Velaptor.Observables
+{
+    // ReSharper disable RedundantNameQualifier
+    using System.Diagnostics.CodeAnalysis;
+    using Velaptor.Observables.Core;
+    using Velaptor.Observables.ObservableData;
+
+    // ReSharper restore RedundantNameQualifier
+
+    /// <summary>
+    /// Creates an observable to send push notifications of OpenGL events.
+    /// </summary>
+    internal class OpenGLInitReactor : Reactor<GLInitData>
+    {
+        /// <summary>
+        /// Sends a push notification that OpenGL has been initialized.
+        /// </summary>
+        /// <param name="data">The data to send with the notification.</param>
+        [SuppressMessage(
+            "ReSharper",
+            "ForCanBeConvertedToForeach",
+            Justification = "Required for proper observable operation.")]
+        public override void PushNotification(GLInitData data)
```

Comment:
Add new parameter called `bool unsubscribeAfterProcessing` with a default value of false

Do this for all `Reactor` types in code base